python/paddle/fluid/tests/unittests/test_adamw_op_static.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import unittest
+import numpy as np
+from paddle.fluid import core
+import paddle.fluid as fluid
+import paddle
+
+
+class testAdamWOpStatic(unittest.TestCase):
```

Comment:
the unitest is tooooo simple !!!
you should take python/paddle/fluid/tests/unittests/test_adam_op.py as a reference, and AT LEAST has a unitest for accuracy. for example build a simple small network + feed same data, then compare the 5th-step-loss with paddle.optimizer.adamw, which should be exactly the same.

Reply:
A unittest for accuracy has been added.